docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       - PGHOST=localhost
       - PGDATABASE=postgis
     ports:
-      - "5432:5432"
+      - "5439:5432"
```

Comment:
Hope this is ok, Basically using 5432 was in conflict with my local pg instance

Reply:
this is fine, we'll need to update some things in the scripts directory to use this port though